src/docker-build-push.js

```diff
@@ -20,7 +20,7 @@ const processBuildArgsInput = buildArgsInput => {
 };
 
 const splitTags = stringTags =>
-  stringTags === null || stringTags === undefined ? undefined : stringTags.split(',').map(tag => tag.trim());
+  stringTags === null || stringTags === undefined || stringTags === '' ? undefined : stringTags.split(',').map(tag => tag.trim());
```

Comment:
Please fix the `prettier` error here. Should appear in multiple lines, like this:

```
const splitTags = stringTags =>
  stringTags === null || stringTags === undefined || stringTags === ''
    ? undefined
    : stringTags.split(',').map(tag => tag.trim());
```

Reply:
Done